sky/shell/platform/ios/sky_dynamic_service_loader.mm

```diff
@@ -156,18 +182,18 @@ - (SkyDynamicServiceHandler)serviceEntryPoint {
   }
 
   dlerror();
-  void* entry = dlsym(_libraryHandle, kFlutterServiceMainFunctionName);
+  void* entry = dlsym(_libraryHandle, kFlutterServiceInvokeProcName);
 
   if (entry == NULL || dlerror() != NULL) {
     LOG(INFO) << "Could not find service entry point '"
```

Comment:
Yeah, had the same feeling. This file used to be so simple :( Will separate this out.